### PR TITLE
Move lunches, alloc more space for coffee breaks

### PR DESCRIPTION
**Pull request by ziotom78**
The present schedule foresees lunches at 11:45, which is a bit early (not sure the canteen is already open). This PR enlarges coffee breaks and pushes lunches 30 minutes later. Afternoon breaks are shifted later as well.

### CODE REVIEW

**Comment by keskitalo** [comment]
This looks fine to me. We had originally set the end time for Friday to be early, anticipating that people would be catching trains and flights. @tskisner @zonca ?

**Comment by zonca** [comment]
I think 15 minutes coffee breaks are better, we are going to have coffee delivered close to the room right?

File: README.md [hunk 1]
```diff
@@ -34,14 +34,14 @@ If you get any error logging in, please email `consult@nersc.gov` with details a
...
-17:00 | Hackathon Ideas and Discussion
-17:30 | End of Day 1
+13:30 | Arrive and Set Up
+14:00 | Meeting Overview
+14:15 | Introduction to TOAST
+15:15 | Simulated Scan Strategies
+16:15 | Coffee Break
+16:45 | Simulated Sky Signals
+17:45 | Hackathon Ideas and Discussion
+18:45 | End of Day 1
```
Comment (zonca) [inline README.md]: shorten coffee break, finish by 18:30?

File: README.md [hunk 2]
```diff
@@ -66,13 +66,13 @@ Time | Event
...
-13:15 | More Hacking
-14:30 | Coffee Break
-14:45 | More Hacking
-15:30 | Close out
-16:00 | End of Day 3
+13:30 | More Hacking
+15:30 | Coffee Break
+16:00 | More Hacking
+17:00 | Close out
+17:15 | End of Day 3
```
Comment (zonca) [inline README.md]: do coffee break at 3-3:15pm and end day at 4:30pm? I think people are going to leave early, I myself have train at 5pm

Comment (ziotom78) [reply README.md]: I didn't think about people leaving earlier, you're right.

File: README.md (hunk 1 above)
Comment (ziotom78) [reply README.md]: Ok, fine for me, when I changed all of them I was aiming for consistency.

**Pull request merged**